app/misc/git.ts

```diff
@@ -810,12 +841,12 @@ function displayModifiedFiles() {
         if (modifiedFiles.length !== 0) {
           if (document.getElementById("modified-files-message") !== null) {
             let filePanelMessage = document.getElementById("modified-files-message");
-            filePanelMessage.parentNode.removeChild(filePanelMessage);
+            filePanelMessage.parentNode.Child(filePanelMessage);
```

Comment:
I'm just curious about this, what necessitates this change?

Reply:
I changed this to test during the functionality of removing nodes. There wasn't any effect by the change here, but I forgot to change it back. I'll change it right now and update the files.